src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,151 @@
+.NavbarItem {
+  background: linear-gradient(90deg, rgb(8, 8, 8) 0%, rgb(77, 77, 77) 100%);
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.navbar-logo {
```

Comment:
użyj konwencji BEM do nazw klas np: .nav-bar__logo

https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

Reply:
Done. Nie wiem czy do końca to czuje ale się starałem żeby było zgodnie z konwencja BEM.